src/ssh_client.py

```diff
@@ -0,0 +1,27 @@
+import paramiko
+
+
+class SSHClient:
+    def __init__(self, host, username, password):
+        self.client = paramiko.SSHClient()
+        self.client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.client.connect(host, username=username, password=password)
+
+    def upload_file(self, local_path, remote_path):
+        sftp = self.client.open_sftp()
+        sftp.put(local_path, remote_path)
+        sftp.close()
+
+    def execute_command(self, command):
+        stdin, stdout, stderr = self.client.exec_command(command)
+        return stdout.read().decode('utf-8') + stderr.read().decode('utf-8')
```

Comment:
better to return tuple
```suggestion
        return stdout.read().decode('utf-8') , stderr.read().decode('utf-8')
```